resources/api/base.py

```diff
@@ -47,15 +47,12 @@ def to_representation(self, obj):
             for lang in LANGUAGES:
                 key = "%s_%s" % (field_name, lang)
                 val = getattr(obj, key, None)
-                if val is None:
+                if val is None or val == "":
                     continue
                 d[lang] = val
 
             # If no text provided, leave the field as null
-            for key, val in d.items():
-                if val is not None:
-                    break
-            else:
+            if not d:
```

Comment:
```python
d = d or None  # if value is falsy, let value be null
```
(same thing, though makes it more obvious that (False, (), [], {}, 0, 0.0, ""), etc. are converted)